### PR TITLE
crypto: se050: Foundries.io Plug-and-Trust update to Release 0.0.2

### PR DESCRIPTION
**Pull request by ldts**
Track NXP's quaterly Plug-and-Trust updates.

These changes are required to build the latest release .
https://github.com/foundriesio/plug-and-trust

The release
https://github.com/foundriesio/plug-and-trust/tree/v0.0.2

Signed-off-by: Jorge Ramirez-Ortiz <jorge@foundries.io>

### CODE REVIEW

File: core/drivers/crypto/se050/adaptors/sub.mk [hunk 1]
```diff
@@ -1,5 +1,6 @@
 cflags-y += -Wno-strict-aliasing
 cflags-y += -DAX_EMBEDDED=1
+cflags-y += -DUSE_RTOS=0
```
Comment (jforissier) [inline core/drivers/crypto/se050/adaptors/sub.mk:3]: Would it be worth moving the common flags to a common file? For example create `core/drivers/crypto/se050/cflags.mk` and here or there `include cflags.mk` or `include ../cflags.mk`. Then you only need to update `cflags.mk`.

Comment (ldts) [reply core/drivers/crypto/se050/adaptors/sub.mk:3]: sounds good, will do that.
will also do the release.tarball, modify shippable and update the PR accordingly, better that way

**Comment by jforissier** [comment]
- About commit: "crypto: se050: Foundries Plug-and-Trust Release 0.0.2"
It does two things, I'd rather have one commit to move the common flags to `cflags.mk` first
- About commit: ".shippable: se050: Foundries Plug-and-Trust 0.0.2 update":
I know I'm being picky :wink: but would you mind cherry-picking this commit first: https://github.com/jforissier/optee_os/commit/3ccd47a8063e1f6fd6e571882bf9bc6a40b86df3 (and applying your Ack if it looks good to you), then the version upgrade is further simplified.

With that you can add:
`Reviewed-by: Jerome Forissier <jerome@forissier.org>`

**Comment by ldts** [comment]
yeah completely agree (thanks for your commit...I was hit with that same problem this morning!)

**Comment by ldts** [comment]
> yeah completely agree (thanks for your commit...I was hit with that same problem this morning!)

yeah it took me three shippables to notice :)

**Comment by ldts** [comment]
@jforissier updated.

**Pull request merged**